module/commands/help.py

```diff
@@ -31,80 +88,106 @@ def help_cmd(update: Update, context: CallbackContext):
+    
+def help_projects_acknowledgements(update: Update, context: CallbackContext, chat_id: int, message_id: int):
+    """Called by the sm_help_reg button from the /help command.
+    Lists to the user the commands related to the other's project and acknowledgements
 
-    keyboard.append([
-        InlineKeyboardButton("📂 Drive",             callback_data="md_drive"),
-        InlineKeyboardButton("📂 GitLab",            callback_data="md_gitlab")
-    ])
+    Args:
+        update: update event
+        context: context passed by the handler
+        chat_id: id of the chat the command was invoked from
+        message_id: id of the help message
+    """
+    message_text = "Visualizzazione comandi relativi a:"
 
+    keyboard = [[]]
     keyboard.append([InlineKeyboardButton(" ~ Progetti e Riconoscimenti ~ ", callback_data="NONE")])
-
+    
     keyboard.append([
         InlineKeyboardButton("📈 Opis Manager",      callback_data="md_opismanager"),
         InlineKeyboardButton("Contributors",         callback_data="md_contributors")
     ])
-
-    keyboard.append([
-        InlineKeyboardButton("Tutti i comandi", callback_data="md_help"),
-        InlineKeyboardButton("Chiudi",          callback_data="exit_cmd")
-    ])
-
     reply_markup = InlineKeyboardMarkup(keyboard)
 
-    context.bot.sendMessage(chat_id=chat_id, text=message_text, reply_markup=reply_markup)
+    context.bot.editMessageText(text=message_text, chat_id=chat_id, message_id=message_id, reply_markup=reply_markup)
 
-def rapp_menu(update: Update, context: CallbackContext, chat_id: int, message_id: int):
-    """Called by the sm_rapp_menu button from the /help command.
-    Allows the user to select the department
+def help_misc(update: Update, context: CallbackContext, chat_id: int, message_id: int):
+    """Called by the sm_help_reg button from the /help command.
+    Lists to the user the commands related to the miscellaneous stuff
 
     Args:
         update: update event
         context: context passed by the handler
         chat_id: id of the chat the command was invoked from
         message_id: id of the help message
     """
-    message_text = "Quali rappresentanti vuoi contattare?"
+    message_text = "Visualizzazione comandi relativi a:"
 
     keyboard = [[]]
-    keyboard.append(
-        [
-            InlineKeyboardButton("Rapp. DMI",         callback_data="md_rappresentanti_dmi"),
-            InlineKeyboardButton("Rapp. Informatica", callback_data="md_rappresentanti_informatica"),
-            InlineKeyboardButton("Rapp. Matematica",  callback_data="md_rappresentanti_matematica"),
-        ]
-    )
+    keyboard.append([InlineKeyboardButton(" ~ Bot e varie ~ ", callback_data="NONE")])
+    
+    keyboard.append([
+        InlineKeyboardButton("📂 Drive",             callback_data="md_drive"),
+        InlineKeyboardButton("📂 GitLab",            callback_data="md_gitlab")
+    ])
```

Comment:
is there a back button to get the main menu again?

Reply:
Planned for the incoming PR regarding #122